multicluster/charts/linkerd-multicluster/templates/gateway.yaml

```diff
@@ -34,10 +34,10 @@ spec:
       labels:
         app: {{.Values.gateway.name}}
     spec:
-      {{- if .Values.enablePodAntiAffinity -}}
       {{- $local := dict "label" "app" "component" .Values.gateway.name -}}
-      {{- include "linkerd.pod-affinity" $local | nindent 6 -}}
-      {{- end }}
+      {{- $_ := set $tree "component" .Values.gateway.name -}}
+      {{- $_ := set $tree "label" "app" -}}
+      {{- include "linkerd.affinity" $tree | nindent 6 }}
```

Comment:
Can you add the `nodeAffinity` section into this chart's values.yaml file?

Reply:
Done

Reply:
Done